src/txn/txn_rollback_to_stable.c

```diff
@@ -91,7 +93,23 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
         if (upd->txnid == WT_TXN_ABORTED)
             continue;
 
-        if (rollback_timestamp < upd->durable_ts || upd->prepare_state == WT_PREPARE_INPROGRESS) {
+        /*
+         * The unstable update according to the following needs to be aborted.
```

Comment:
I suggest: "An unstable update needs to be aborted if any of the following are true:"

Reply:
Updated as per the suggestion.